sdk/cosmosdb/cosmos/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "extends": "../../../tsconfig.package",
   "compilerOptions": {
     "module": "esnext",
-    "strict": false,
+    "strict": true,
```

Comment:
could you remove the helpers file from the list in `tsconfig.strict.json` instead?

Reply:
yes! I can do this